docs/user/guides/setup/configuration-file.rst

```diff
@@ -0,0 +1,223 @@
+How to add a configuration file
+===============================
+
+As part of the initial setup for your Read the Docs site,
+you need to create a **configuration file** called ``.readthedocs.yaml``.
+The configuration file tells Read the Docs what specific settings to use for your project.
+
+This how-to guide covers:
+
+#. Where to put your configuration file.
+#. Templates to setup your configuration file from scratch.
+#. The most basic sections in the configuration file that can help you get started.
+#. Next steps...
```

Comment:
This summary could use some condensing/clarification, so that it's more user-centric and the user knows exactly what they are getting from this guide:

```suggestion
#. Where to put your configuration file.
#. What to put in the configuration file.
#. How to customize the configuration for your project.
```

There is nuance there from our side of course, but this is what I'd care about as a user reading this guide

Reply:
Sure - the previous list was a reflection of the section structure. Not a big loss, though.